recipes/assetid/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "assetid" %}
+{% set version = "0.0.4" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_rev: {{ version }}
+  git_url: "https://github.com/axiom-data-science/{{ name }}"
```

Comment:
> Is there any way to get that sha without downloading from Github (i.e. can this be reproduced)?

Sadly no. You need a trust download to compute the `sha`.

> The reason I used a tag was because I imagine Github could change this (e.g. compression level) and it may break.

That won't happen as we are pointing to the tarball. Unless you delete and re-issue that release. In that case it should break to warn us that the source has changed.